.github/workflows/playwright.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           npx wp-env run cli "wp --info"
       - name: Activate performance experiments
         run: |
-          npx wp-env run cli "wp elementor experiments activate e_font_icon_svg,e_lazyload,e_dom_optimization,e_optimized_assets_loading,e_optimized_css_loading,additional_custom_breakpoints"
+          npx wp-env run cli "wp elementor experiments activate e_font_icon_svg,e_lazyload,e_dom_optimization,e_optimized_assets_loading,e_optimized_css_loading,additional_custom_breakpoints --skip-themes --skip-plugins=woocommerce"
```

Comment:
Skip iterating through hooks for themes and woocommerce hooks when activating experiments.

Reply:
Execution time before:
![image](https://github.com/elementor/elementor/assets/130742176/4638b1ba-3649-4b77-bbb4-224332f3461b)

Reply:
Execution time with skip themes and plugins 2s 539ms
![image](https://github.com/elementor/elementor/assets/130742176/c2b69407-544b-4c11-9185-02d0f1c43ee6)

Reply:
Execution time with skip themes only 2s 382ms
![image](https://github.com/elementor/elementor/assets/130742176/fbe37112-ec87-4072-81ac-2fc374733766)

Reply:
Execution time with skip Woocommerce only 
![image](https://github.com/elementor/elementor/assets/130742176/4dbca124-8909-444c-a115-d0b709c6cee7)